src/Features/Core/Portable/RemoveUnusedMembers/AbstractRemoveUnusedMembersDiagnosticAnalyzer.cs

```diff
@@ -450,6 +450,9 @@ private bool IsCandidateSymbol(ISymbol memberSymbol)
                         case SymbolKind.Property:
                             return ((IPropertySymbol)memberSymbol).ExplicitInterfaceImplementations.IsEmpty;
```

Comment:
why are we not checking virtual/abstract/etc. here?

Reply:
There is a check up front to only operate on private member symbols.

Reply:
then why do we check that stuff for methods?  Thanks!

Reply:
Seems like an oversight, let me delete that code.